reactive/kotlinx-coroutines-reactive/src/Await.kt

```diff
@@ -169,6 +188,8 @@ private suspend fun <T> Publisher<T>.awaitOne(
 
         @Suppress("UNCHECKED_CAST")
         override fun onComplete() {
+            subscription.checkInitialized("onComplete") // TODO: maybe don't enforce?
```

Comment:
It seems like other implementations from reactive libraries do not enforce such rule, maybe we could stick with that as well